src/java/org/httpkit/HttpUtils.java

```diff
@@ -257,6 +257,14 @@ public static String getHost(URI uri) {
         }
         return host;
     }
+    
+    public static String getProxyHost(URI uri){
```

Comment:
It looks like they default to just HTTP https://github.com/dakrone/clj-http/blob/master/src/clj_http/core.clj#L117. Unless they modify the scheme of the host later on, but they aren't using the 3 arity form of HttpHost that allows you to set the scheme. https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/HttpHost.html